packages/valory/contracts/mech_marketplace/contract.py

```diff
@@ -73,6 +82,11 @@ class MechOperation(Enum):
     DELEGATE_CALL = 1
 
 
+def pad_address_for_topic(address: str) -> HexBytes:
+    """Left-pad an Ethereum address to 32 bytes for use in a topic."""
+    return HexBytes(Ox + address[Ox_CHARS:].zfill(TOPIC_CHARS))
```

Comment:
Is this or the constants used anywhere?

Reply:
This was added in your PR, so I took it directly from core.